scripts/persistence/notion/catalog.py

```diff
@@ -0,0 +1,29 @@
+"""Models for the notion service catalog https://www.notion.so/localstack/3c0f615e7ffc4ae2a034f1ed9c444bd2"""
+
+from notion_client import Client as NotionClient
+
+from notion_objects import (
+    Page,
+    TitlePlainText,
+    Status,
+    Database,
+    Checkbox,
+    PeopleProperty,
+    Text
+)
+
+DEFAULT_CATALOG_DATABASE_ID = "3c0f615e7ffc4ae2a034f1ed9c444bd2"
+
+
+class PersistenceServiceItem(Page):
+    name = TitlePlainText("Name")
+    status = Status("Persistence")
+    has_test = Checkbox("Persistence Tests")
+    primary_owner = PeopleProperty("Primary Owner")
+    secondary_owner = PeopleProperty("Secondary Owner(s)")
+    limitations = Text("Limitations") 
```

Comment:
To address @joe4dev's comment, maybe we could rename this column to something like `Limitations (synced with docs)`?